_data/projects/bitwarden-clients.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: Bitwarden Clients
+desc: >-
+  Bitwarden client applications (web, browser extension, desktop, and cli).
```

Comment:
This field is too long, and I'm honestly not sure it won't break the site. For reference, most projects only fill out a sentence or two here:

<img width="763" src="https://user-images.githubusercontent.com/359239/197282901-90d34291-63ab-4e5b-b2cb-f4c2da652245.png">

Reply:
Nevermind, you can see it on staging:

<img width="1555"  src="https://user-images.githubusercontent.com/359239/197283178-81290b54-b708-4f75-91dd-899c83dc5bd0.png">

Can we shrink this down a bit?